groupAndroidPackages.json

```diff
@@ -98,6 +98,18 @@
         "^com\\.willowtreeapps\\.hyperion:"
       ]
     },
+    {
+      "groupName": "Jetpack Compose",
+      "packagePatterns": [
+        "^androidx.\\.compose:",
+        "^androidx.\\.compose\\.animation:",
+        "^androidx.\\.compose\\:compiler",
+        "^androidx.\\.compose\\.foundation:",
+        "^androidx.\\.compose\\.material:",
+        "^androidx.\\.compose\\.runtime:",
+        "^androidx.\\.compose\\.ui:"
```

Comment:
Confirmed that these cover current all compose libraries: https://developer.android.com/jetpack/androidx/explorer

![image](https://user-images.githubusercontent.com/909674/106223252-65ba8780-6224-11eb-9caf-810b06294a68.png)

---

```suggestion
        "^androidx\\.compose:",
        "^androidx\\.compose\\.animation:",
        "^androidx\\.compose\\:compiler",
        "^androidx\\.compose\\.foundation:",
        "^androidx\\.compose\\.material:",
        "^androidx\\.compose\\.runtime:",
        "^androidx\\.compose\\.ui:"
```